Assets/Treasure/TDK/Runtime/Services/Thirdweb/TDKThirdwebService.cs

```diff
@@ -134,6 +164,54 @@ public async Task ConnectWallet(EcosystemWalletOptions ecosystemWalletOptions, i
             }
         }
 
+        public async Task ConnectExternalWallet(int chainId)
+        {
+            _connectionCancelationTokenSource?.Cancel();
+            // TODO where to get supported chains list?
```

Comment:
This question still stands, I suppose its for chain switching? I just set the active chain here, assuming its not going to be common to switch chains in game but let me know if thats not correct

Reply:
Active chain should be good for now. Does the `1` need to be there too?

Reply:
Probably not, i'll remove it